tfconfig/load.go

```diff
@@ -44,6 +45,15 @@ func IsModuleDir(dir string) bool {
 	return true
 }
 
+func contains(providers []*ProviderRef, provider ProviderRef) bool {
```

Comment:
Could you rename this function (or better yet, add the `ProviderRef` type as a receiver). The name is very generic. Maybe something like:

```golang
func (provider *ProviderRef) in(providers []*ProviderRef) bool {
```